assets/dash.css

```diff
@@ -0,0 +1,60 @@
+#header {
+  background-color: #000 !important;
```

Comment:
Curious: `!important` is a code smell from the days I last CSS'd. Which selector's more specific than #id?

Reply:
I agree re: smell but bootstrap is still full of `!important`. Not sure off the top of my head what was overriding this, but it was something `!important` somewhere else.